s/nub-keyboard/utils/dispatchCustomEvent.ts

```diff
@@ -0,0 +1,9 @@
+export const dispatchCustomEvent = (element: HTMLElement, data: any) => {
```

Comment:
this is an interesting technique to dispatch custom events,

though we should probably formalize each custom event class, and make them available for developers.